packages/rocketchat-ui-sidenav/client/roomList.js

```diff
@@ -19,17 +19,16 @@ Template.roomList.helpers({
 		const query = {
 			open: true
 		};
-		let sort = { 't': 1, 'name': 1 };
+		const sort = { 't': 1, 'name': 1 };
 		if (this.identifier === 'f') {
 			query.f = favoritesEnabled;
 		} else {
 			let types = [this.identifier];
 			if (this.identifier === 'activity') {
 				types = ['c', 'p', 'd'];
-				sort = { lastActivity : -1, _updatedAt: -1};
```

Comment:
according to [this](https://github.com/RocketChat/Rocket.Chat/commit/52b43d6a1f01a1fb56c830e5cfa0de4eef445460) it should still sort by `_updatedAt`

Reply:
I agree with @sampaiodiego

Reply:
@sampaiodiego @engelgabriel `sort` is not used if the `indentifier === "activity"`